doc/sphinx-guides/source/developers/dataset-semantic-metadata-api.rst

```diff
@@ -0,0 +1,91 @@
+
+
+Add Dataset Metadata
+--------------------
+
+To add json-ld formatted metadata for a Dataset, specify the Dataset ID (DATASET_ID) or Persistent identifier (DATASET_PID). Adding '?replace=true' will overwrite an existing metadata value. The default (replace=false) will only add new metadata or add a new value to a multi-valued field. 
+
+.. code-block:: bash
+
+  export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+  export DATASET_ID='12345'
+  export DATASET_PID='doi:10.5072/FK2A1B2C3'
+  export VERSION='1.0'
+  export SERVER_URL=https://demo.dataverse.org
+ 
+  Example: Change the Dataset title 
+ 
+    curl -X PUT -H X-Dataverse-key:$API_TOKEN -H 'Content-Type: application/ld+json' -d '{"Title": "Submit menu test", "@context":{"Title": "http://purl.org/dc/terms/title"}}' "$SERVER_URL/api/datasets/$DATASET_ID/metadata?replace=true"
+
+  Example 2: Add a description using the DATASET PID
+
+    curl -X PUT -H X-Dataverse-key:$API_TOKEN -H 'Content-Type: application/ld+json' -d '{"citation:Description": {"dsDescription:Text": "New description"}, "@context":{"citation": "https://dataverse.org/schema/citation/","dsDescription": "https://dataverse.org/schema/citation/dsDescription#"}}' "$SERVER_URL/api/datasets/:persistentId/metadata?persistentId=$DATASET_PID"
+
+You should expect a 200 ("OK") response indicating whether a draft Dataset version was created or an existing draft was updated.
+
+
+Delete Dataset Metadata
+-----------------------
+
+To delete metadata for a Dataset, send a json-ld representation of the fields to delete and specify the Dataset ID (DATASET_ID) or Persistent identifier (DATASET_PID).
+
+.. code-block:: bash
+
+  export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+  export DATASET_ID='12345'
+  export DATASET_PID='doi:10.5072/FK2A1B2C3'
+  export VERSION='1.0'
+  export SERVER_URL=https://demo.dataverse.org
+ 
+  Example: Delete the TermsOfUseAndAccess 'restrictions' value 'No restrictions' for the latest version using the DATASET PID
+
+    curl -X PUT -H X-Dataverse-key:$API_TOKEN -H 'Content-Type: application/ld+json' -d '{"https://dataverse.org/schema/core#restrictions":"No restrictions"}' "$SERVER_URL/api/datasets/:persistentId/metadata/delete?persistentId=$DATASET_PID"
+
+Note, this example uses the term URI directly rather than adding an '@context' element. You can use either form in any of these API calls. 
+
+You should expect a 200 ("OK") response indicating whether a draft Dataset version was created or an existing draft was updated.
+
+
+Create a Dataset
+----------------
+
+Specifying the Content-Type as application/ld+json with the existing /api/dataverses/{id}/datasets API call (see :ref:`create-dataset-command`) supports using the same metadata format when creating a Dataset.
+
+With curl, this is done by adding the following header:
+
+.. code-block:: bash
+
+  -H 'Content-Type: application/ld+json' 
```

Comment:
I'm confused about how this just sort of ends. Can we have a downloadable JSON-LD file to use as an example when creating a dataset?

Reply:
I added an example.